lib/gui/os/window-progress/services/window-progress.js

```diff
@@ -47,11 +44,14 @@ module.exports = function() {
    * OSWindowProgressService.set(85);
    */
   this.set = (percentage) => {
-    if (percentage > 100 || percentage < 0) {
+    const PERCENTAGE_MINIMUM = 0;
+    const PERCENTAGE_MAXIMUM = 100;
+
+    if (percentage > PERCENTAGE_MAXIMUM || percentage < PERCENTAGE_MINIMUM) {
```

Comment:
Only a minor nit, but I think comparing the minimum value first reads more naturally? i.e.
```js
if (percentage < PERCENTAGE_MINIMUM || percentage > PERCENTAGE_MAXIMUM) {
```